### ISSUE TITLE
Package feeds

### ISSUE THREAD

**Issue by tchoutri** [flora-pm/flora-server#329]
An Atom feed will be provided at `/packages/:namespace/:package/feed` where the package authors will be able to subscribe to their package alerts.

Current alerts would include things like:

* A dependency is outdated;
* (more specifically) The package is not compatible with the latest GHC;
* A direct or indirect dependency is affected by a CVE (the publishing of this alert would coincide with the end of any embargo, it's just a matter of getting information to the people more directly).

More ideas welcome!

Depends on #100

**Comment by TixieSalander** [comment]
Maybe releases/versions could also have their own feed (aimed more for users or collaborators) with the changelog as items content for example.

That could take shape into different URLs like `/packages/:namespace/:package/alerts` and `/packages/:namespace/:package/versions` (or `releases`). 

What do you think?

**Comment by tchoutri** [comment]
Yep, release-specific feeds would be included too. Thanks for pointing it out.

**Comment by tchoutri** [comment]
Implemented in #881
